altair/utils/mimebundle.py

```diff
@@ -44,9 +44,15 @@ def spec_to_mimebundle(
     ----
     The png, svg, pdf, and vega outputs require the altair_saver package
     """
+    from altair.utils.display import compile_with_vegafusion, using_vegafusion
```

Comment:
See my comment on the import in `save.py`

Reply:
Done in 6e4c8c4fc3d93e8cbd9e296ad79c11b957729eeb